librz/core/cmd/cmd_help.c

```diff
@@ -619,12 +620,12 @@ RZ_IPI int rz_cmd_help(void *data, const char *input) {
 				d = -d;
 			}
 			if (*input == 'j') {
-				pj_ks(pj, "fvalue", sdb_fmt("%.1lf", core->num->fvalue));
-				pj_ks(pj, "float", sdb_fmt("%ff", f));
-				pj_ks(pj, "double", sdb_fmt("%lf", d));
-				pj_ks(pj, "binary", sdb_fmt("0b%s", out));
+				pj_ks(pj, "fvalue", rz_strf(number, "%.1lf", core->num->fvalue));
+				pj_ks(pj, "float", rz_strf(number, "%ff", f));
+				pj_ks(pj, "double", rz_strf(number, "%lf", d));
```

Comment:
@XVilka  why not instead of a string using directly their pj counterparts?
```c
pj_ks(pj, "fvalue", core->num->fvalue);
pj_ks(pj, "double", d);
pj_ks(pj, "float", f);
```
or is a bad idea since the values are expected to have a certain length, etc.. ?

Reply:
Good point :D I have no idea.